<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_SelectTwoPoints</class>
 <widget class="QMainWindow" name="MainWindow_SelectTwoPoints">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QVTKWidget" name="m_pQVTKWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>480</width>
           <height>540</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>380</width>
           <height>440</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Point 1</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>90</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Point 2</string>
          </property>
         </widget>
         <widget class="QFrame" name="frame_3">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>50</y>
            <width>301</width>
            <height>40</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QDoubleSpinBox" name="m_pDoubleSpinBoxP1X">
           <property name="geometry">
            <rect>
             <x>39</x>
             <y>10</y>
             <width>61</width>
             <height>20</height>
            </rect>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <double>-1000000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>16</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>X</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>10</y>
             <width>16</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>10</y>
             <width>16</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Z</string>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="m_pDoubleSpinBoxP1Y">
           <property name="geometry">
            <rect>
             <x>134</x>
             <y>10</y>
             <width>61</width>
             <height>20</height>
            </rect>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <double>-1000000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="m_pDoubleSpinBoxP1Z">
           <property name="geometry">
            <rect>
             <x>230</x>
             <y>10</y>
             <width>61</width>
             <height>20</height>
            </rect>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <double>-1000000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </widget>
         <widget class="QFrame" name="frame_4">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>120</y>
            <width>301</width>
            <height>40</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QDoubleSpinBox" name="m_pDoubleSpinBoxP2X">
           <property name="geometry">
            <rect>
             <x>39</x>
             <y>10</y>
             <width>61</width>
             <height>20</height>
            </rect>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <double>-1000000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>16</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>X</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_11">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>10</y>
             <width>16</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_12">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>10</y>
             <width>16</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Z</string>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="m_pDoubleSpinBoxP2Y">
           <property name="geometry">
            <rect>
             <x>134</x>
             <y>10</y>
             <width>61</width>
             <height>20</height>
            </rect>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <double>-1000000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="m_pDoubleSpinBoxP2Z">
           <property name="geometry">
            <rect>
             <x>230</x>
             <y>10</y>
             <width>61</width>
             <height>20</height>
            </rect>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <double>-1000000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>180</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Radiuses</string>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="m_pDoubleSpinBoxRadiuses">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>180</y>
            <width>61</width>
            <height>21</height>
           </rect>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <double>-1000000000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1046</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadFile"/>
    <addaction name="actionSaveFile"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoadFile">
   <property name="text">
    <string>Load File</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>Save File</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvtkwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
